doc/_ext/ceph_commands.py

```diff
@@ -156,31 +154,37 @@ def is_reasonably_simple(self):
 
     def mk_bash_example(self):
         simple = self.is_reasonably_simple()
-        line = ' '.join(['ceph', self.prefix()] + [p.mk_bash_example(simple) for p in self.params])
+        line = ' '.join(['ceph', self.prefix] + [p.mk_bash_example(simple) for p in self.params])
         return line
 
 
 class Sig:
     @staticmethod
-    def _param_to_sig(p):
+    def _parse_arg_desc(desc):
         try:
-            return {kv.split('=')[0]: kv.split('=')[1] for kv in p.split(',') if kv}
+            return {kv.split('=')[0]: kv.split('=')[1] for kv in desc.split(',') if kv}
```

Comment:
If I'm not wrong, tuples are also valid for dict comprehensiones.

```suggestion
            return {kv.split('=') for kv in desc.split(',') if kv}
```

How strict is this command prefix syntax? Whitespaces are not allowed?: `k1=v1, k2 = v2 , ...`?

Reply:
@epuertat we do have a use case where there are space(s) after ",". see https://github.com/ceph/ceph/blob/72cc819b3aa47cdb684577be1f33021aa9fd2faf/src/mon/MonCommands.h#L515-L523

we could enforce the policy that only a single "," is allowed for the delimiter between parameter properties. but that's not in the scope of this PR. so i'd defer it to another change.